### ISSUE TITLE
Enable oled motion pro on LG C2?

### ISSUE THREAD

**Issue by Sevylox** [Maassoft/ColorControl#135]
Is this setting supposed to actually work on some of the 2022 tvs? When I enable it none of the options do anything except at high and 60hz, which is what the tv was already capable of natively. Only thing that happens weirdly is that the picture gets brighter as I lower the setting.

**Comment by Maassoft** [comment]
No, I'm afraid not. It was added for users to test it on the 2022 TV's, but I haven't heard it working like it did on the 2019 series. I'll probably just better remove it for the 2022 TV's to avoid confusion.

**Comment by Sevylox** [comment]
I looked into it more and it mostly has to do with BFI being broken on a 120hz signal for LG C2. Maybe in a future fix that allows strobing at 60hz even on a 120hz signal this hack could potentially work.
